src/oled_display_node.cpp

```diff
@@ -55,17 +55,18 @@ int main(int argc, char** argv) {
 
 // Default lines to output text to from this module
 #define  DISP_LINE_HOSTNAME    0
-#define  DISP_LINE_IP_ADDR     1
-#define  DISP_LINE_BATT_VOLTS  3
-#define  DISP_LINE_MOTOR_POWER 5 
+#define  DISP_LINE_IP_WLAN     1
+#define  DISP_LINE_IP_ETH     3
+#define  DISP_LINE_BATT_VOLTS  5
+#define  DISP_LINE_MOTOR_POWER 7 
 
 // We are putting a battery low blinking feature to warn user of very low battery
 // Our goal is to set a lower threshold than this (default 22.5V) in motor node to stop motor control
 // once that point is hit
 #define  BAT_LOW_LEVEL  23.5                     // Start to warn of low battery voltage with blinking display
 
 // Define a level where we think the charger is plugged in at this time
-#define  BAT_CHARGING_LEVEL  27.0                // Indicate battery is on charger for high voltages
+#define  BAT_CHARGING_LEVEL  27.8                // Indicate battery is on charger for high voltages
```

Comment:
I have noticed this.  It needs to be above 27, just how much is the question.   I would have picked 12.5 but basically there is no 'best' answer so 27.8 is ok here.